src/runtime/go.mod

```diff
@@ -0,0 +1,62 @@
+
+go 1.14
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/blang/semver v0.0.0-20190414102917-ba2c2ddd8906
+	github.com/cilium/ebpf v0.0.0-20200421083123-d05ecd062fb1 // indirect
+	github.com/containerd/cgroups v0.0.0-20190717030353-c4b9ac5c7601
+	github.com/containerd/console v0.0.0-20191206165004-02ecf6a7291e
+	github.com/containerd/containerd v1.2.1-0.20181210191522-f05672357f56
+	github.com/containerd/cri-containerd v1.11.1-0.20190125013620-4dd6735020f5
+	github.com/containerd/fifo v0.0.0-20190226154929-a9fb20d87448
+	github.com/containerd/go-runc v0.0.0-20200220073739-7016d3ce2328 // indirect
+	github.com/containerd/ttrpc v1.0.0 // indirect
+	github.com/containerd/typeurl v1.0.1-0.20190228175220-2a93cfde8c20
+	github.com/containernetworking/plugins v0.8.2
+	github.com/cri-o/cri-o v1.0.0-rc2.0.20170928185954-3394b3b2d6af
+	github.com/dlespiau/covertool v0.0.0-20180314162135-b0c4c6d0583a
+	github.com/docker/go-units v0.3.3
+	github.com/go-ini/ini v1.28.2
+	github.com/go-openapi/errors v0.18.0
+	github.com/go-openapi/runtime v0.18.0
+	github.com/go-openapi/strfmt v0.18.0
+	github.com/go-openapi/swag v0.18.0
+	github.com/go-openapi/validate v0.18.0
+	github.com/gogo/protobuf v1.2.0
+	github.com/grpc-ecosystem/grpc-opentracing v0.0.0-20180507213350-8e809c8a8645 // indirect
+	github.com/hashicorp/go-multierror v1.0.0
+	github.com/hashicorp/yamux v0.0.0-20190923154419-df201c70410d // indirect
+	github.com/intel/govmm v0.0.0-20200304142514-e969afbec52c
+	github.com/kata-containers/agent v0.0.0-20200220202609-d26a505efd33
+	github.com/kata-containers/runtime v0.0.0-00010101000000-000000000000
+	github.com/mdlayher/vsock v0.0.0-20191108225356-d9c65923cb8f // indirect
+	github.com/mitchellh/mapstructure v1.1.2
+	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
+	github.com/opencontainers/runc v1.0.0-rc9.0.20200102164712-2b52db75279c
+	github.com/opencontainers/runtime-spec v1.0.2-0.20190408193819-a1b50f621a48
+	github.com/opencontainers/selinux v1.4.0
+	github.com/opentracing/opentracing-go v1.1.0
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/procfs v0.0.0-20190328153300-af7bedc223fb
+	github.com/safchain/ethtool v0.0.0-20190326074333-42ed695e3de8
+	github.com/seccomp/libseccomp-golang v0.9.1 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/stretchr/testify v1.4.0
+	github.com/uber/jaeger-client-go v0.0.0-20200422204034-e1cd868603cb
+	github.com/uber/jaeger-lib v2.2.0+incompatible // indirect
+	github.com/urfave/cli v1.20.1-0.20170926034118-ac249472b7de
+	github.com/vishvananda/netlink v1.0.1-0.20190604022042-c8c507c80ea2
+	github.com/vishvananda/netns v0.0.0-20180720170159-13995c7128cc
+	go.uber.org/atomic v1.6.0 // indirect
+	golang.org/x/net v0.0.0-20191108221443-4ba9e2ef068c
+	golang.org/x/oauth2 v0.0.0-20191122200657-5d9234df094c
+	golang.org/x/sys v0.0.0-20200420163511-1957bb5e6d1f
+	google.golang.org/grpc v1.19.0
+)
+
+replace (
+	github.com/kata-containers/runtime => ./
```

Comment:
Neat! However, I think there may be a small amount of additional "fix-up" to do:

```sh
$ find . -type f -exec grep -l github.com/kata-containers/runtime {} \;|egrep -v "(go\.mod|\.go)"
./pkg/katatestutils/README.md
./.ci/versions_checker.sh
./.ci/lib.sh
./cli/config/configuration-qemu-virtiofs.toml.in
./cli/config/configuration-fc.toml.in
./cli/config/configuration-qemu.toml.in
./cli/config/configuration-clh.toml.in
./cli/config/configuration-acrn.toml.in
./vendor/modules.txt
./vendor/github.com/kata-containers/runtime/pkg/katatestutils/README.md
./vendor/github.com/kata-containers/runtime/virtcontainers/utils/virtcontainers-setup.sh
./.travis.yml
./README.md
./virtcontainers/utils/virtcontainers-setup.sh
```

Reply:
They are not related to go.mod right? I don't quite understand what `fix-up` you meant.

Reply:
Sorry, what I mean is shouldn't we update all the runtime URLs in files on this PR so that they refer to the *new* runtime URL (in this repo), not the (old) https://github.com/kata-containers/runtime. In effect:

```sh
sed '!https://github.com/kata-containers/runtime!https://github.com/kata-containers/kata-containers/tree/master/src/runtime!g'
```

Reply:
Makes sense! Let's do it in another PR. Sounds OK to you?

Reply:
@bergwolf - ack. I guess it's good to have a "vanilla" import first. btw, I know you've specified the branch point in the PR titl (`1.11.0-rc0`), but I do wonder if it makes sense to create a tag in each stand-alone repo at the point we branch. That would make it 100% clear what we're doing and would also handle the scenario that we're forced to branch at different places for different repos. Ideally the tag name would be identicall across all stand-alone repos.

Also, we are all going to have to be vigilant about "double fixing" (ensuring PRs are applied to the stand-alone repos and this repo). That's going to be particularly important for maintaining the stable branches in the stand-alone repos since we don't want to "miss" fixes that, say, get applied to *this* repos `master`, but not to the stand-alone repos `master`.

Reply:
Sounds good. Any suggestion on the tag name?

Reply:
Tricky. How about `kata-repo-consolidation`?

If we're going to do this, it probably makes sense to create an "annotated tag" in each standalone repo so we can add a brief description including a reference to https://github.com/kata-containers/kata-containers and https://github.com/kata-containers/kata-containers/issues/172.

wdyt @amshinde and @grahamwhaley?